src/docs/reference/modules/Sms/README.md

```diff
@@ -0,0 +1,37 @@
+# SMS (`OrchardCore.Sms`)
+
+This module provides the infrastructure necessary to send messages using `SMS` service.
+
+## SMS Settings
+
+Enabling the `OrchardCore.Sms` will add a new settings under `Configurations` >> `Settings` >> `SMS`. You can utilize these settings to set up the default SMS provider configuration. The following are the providers that are readily accessible.
+
+| Provider | Description |
+| --- | --- |
+| `Console` | This particular provider is exclusively meant for debugging purposes and should never be employed in a production environment. It permits the message to be displayed on the debug console. |
```

Comment:
employed = used or you meant `deployed` ?

Reply:
changed